src/premium/awaitedReactions.md

```diff
@@ -1,26 +1,50 @@
 # Awaited reactions
 *(for premium bots)*
 
-Thanks to awaited reactions your bot can be triggered by reactions
+**Awaited reactions** are same like [awaited commands](../guides/awaitedCommands.md). Unlike awaited commands which waits for a message, it waits for a reaction instead. 
```

Comment:
Should be `Awaited reactions are similar to`, its `wait` not `waits`, and `, they wait for a reaction instead.`

```suggestion
**Awaited reactions** are similar to [awaited commands](../guides/awaitedCommands.md). Unlike awaited commands which wait for a message, they wait for a reaction instead. 
```